index.md

```diff
@@ -5,9 +5,13 @@ layout: default
 <div class="lead pretty-links">
 StarCon is a two-day software engineering conference held at the **University of Waterloo**. Join us for two days of ten minute lighting talks about technology, computer science, programming, and the joy of learning about them!
 
+## Who's speaking? Can I apply to speak?
+
+Check out our [invited speakers](speakers/#keynotes) and [talk proposal details](speakers/#call-for-proposals)!
```

Comment:
I put this here so people know that just because we've invited speakers it doesn't mean they can't apply later

the "details" right now is that we'll have a cfp and it's not out yet